cts/scheduler/year-2038.xml

```diff
@@ -0,0 +1,1201 @@
+<cib crm_feature_set="3.0.14" validate-with="pacemaker-3.0" epoch="104" num_updates="12" admin_epoch="0" cib-last-written="Mon Apr  9 06:28:41 2018" update-origin="controller-0" update-client="cibadmin" update-user="root" have-quorum="1" dc-uuid="2" execution-date="2150000000">
```

Comment:
It should be straightforward to skip the test based on $(getconf LONG_BIT) and/or $(uname -m), but the challenge will be avoiding the "output differs" error.

Reply:
I suppose we could take it out of the "dates" test group and make it the sole member of a new "64bit" test group, so it would have a separate "expected output" file.